src/src/com/microsoft/aad/adal/DateTimeAdapter.java

```diff
@@ -50,12 +50,31 @@
             DateFormat.DEFAULT);
 
     private final DateFormat iso8601Format = buildIso8601Format();
```

Comment:
Users have tokens that are older than 1 year?)) Then do we really care if they will be able to parse date or not? we will remove this token anyway because it's expired
anyway you won't be able to cover all possible date formats that user could set to default, but I think no harm going to be done with adding this couple date formats